pkg/remote/trans/netpoll/http_client_handler_test.go

```diff
@@ -17,48 +17,186 @@
 package netpoll
 
 import (
+	"context"
+	"net/http"
 	"strings"
 	"testing"
+	"time"
 
+	"github.com/cloudwego/kitex/internal/mocks"
+	"github.com/cloudwego/kitex/internal/test"
 	"github.com/cloudwego/kitex/pkg/remote"
+	"github.com/cloudwego/kitex/pkg/remote/trans"
+	"github.com/cloudwego/kitex/pkg/rpcinfo"
+	"github.com/cloudwego/netpoll"
 )
 
+var (
+	cliTransHdlr     remote.ClientTransHandler
+	httpCilTransHdlr remote.ClientTransHandler
+	cilopt           *remote.ClientOption
+)
+
+func init() {
+	cilopt = &remote.ClientOption{
+		SvcInfo: mocks.ServiceInfo(),
+		Codec: &MockCodec{
+			EncodeFunc: nil,
+			DecodeFunc: nil,
+		},
+		Dialer: NewDialer(),
+	}
+
+	cliTransHdlr, _ = NewCliTransHandlerFactory().NewTransHandler(cilopt)
+	httpCilTransHdlr, _ = NewHTTPCliTransHandlerFactory().NewTransHandler(cilopt)
+}
+
+// TestHTTPWrite test http_client_handler Write return err
+func TestHTTPWrite(t *testing.T) {
+	// 1. prepare mock data
+	conn := &MockNetpollConn{}
+	rwTimeout := time.Second
+	cfg := rpcinfo.NewRPCConfig()
+	rpcinfo.AsMutableRPCConfig(cfg).SetReadWriteTimeout(rwTimeout)
+	to := rpcinfo.NewEndpointInfo("", "", nil, map[string]string{
+		rpcinfo.HTTPURL: "https://example.com",
+	})
+	ri := rpcinfo.NewRPCInfo(nil, to, nil, cfg, rpcinfo.NewRPCStats())
+	ctx := context.Background()
+	msg := remote.NewMessage(nil, mocks.ServiceInfo(), ri, remote.Reply, remote.Client)
+
+	// 2. test
+	err := httpCilTransHdlr.Write(ctx, conn, msg)
+	// check err not nil
+	test.Assert(t, err != nil)
+}
+
+// TestHTTPRead test http_client_handler Read return err
+func TestHTTPRead(t *testing.T) {
+	// 1. prepare mock data
+	rwTimeout := time.Second
+
+	var readTimeout time.Duration
+	var isReaderBufReleased bool
+	conn := &MockNetpollConn{
+		SetReadTimeoutFunc: func(timeout time.Duration) (e error) {
+			readTimeout = timeout
+			return nil
+		},
+		ReaderFunc: func() (r netpoll.Reader) {
+			reader := &MockNetpollReader{
+				ReleaseFunc: func() (err error) {
+					isReaderBufReleased = true
+					return nil
+				},
+			}
+			return reader
+		},
+	}
+
+	cfg := rpcinfo.NewRPCConfig()
+	rpcinfo.AsMutableRPCConfig(cfg).SetReadWriteTimeout(rwTimeout)
```

Comment:
sorry I didn't make it clear. You should replace `rpcinfo.AsMutableRPCConfig(cfg).SetReadWriteTimeout(rwTimeout)` with `rpcinfo.AsMutableRPCConfig(cfg).SetRPCTimeout(rwTimeout)`, or rwTimeout won't work.